conf/base.config

```diff
@@ -62,4 +62,7 @@ process {
     withName:CUSTOM_DUMPSOFTWAREVERSIONS {
         cache = false
     }
+    withName:ANTISMASH_ANTISMASHLITE {
+        errorStrategy = { task.exitStatus in [1] ? 'ignore' : task.exitStatus in [143,137,104,134,139] ? 'retry' : 'finish' }
```

Comment:
```suggestion
        errorStrategy = { task.exitStatus == 1 ? 'ignore' : task.exitStatus in [143,137,104,134,139] ? 'retry' : 'finish' }
```

Reply:
I was actually futureproofing in case we have to add additional... or do you think it's safer with `==`?

Reply:
Both are correct and safe. I think, as long as there is only a single comparison my suggested version is clearer but it's nitpicking really. So just keep it, flip a coin, or let Maia decide 😛